lib/maze/hooks/appium_hooks.rb

```diff
@@ -24,15 +24,19 @@ def after_configuration
                                                      tunnel_id,
                                                      config.access_key
         elsif config.farm == :bb
+          if ENV['BUILDKITE']
+            credentials = Maze::BitBarUtils.account_credentials config.tms_uri
+            config.username = credentials[:username]
+            config.access_key = credentials[:access_key]
+          end
           config.app = Maze::BitBarUtils.upload_app config.access_key,
                                                     config.app
           Maze::BitBarUtils.start_local_tunnel config.bb_local,
-                                               config.username,
-                                               config.access_key
+                                              config.username,
+                                              config.access_key
           config.capabilities = Maze::Capabilities.for_bitbar_device config.access_key,
-                                                                     config.device,
-                                                                     config.capabilities_option
-
+                                                                    config.device,
+                                                                    config.capabilities_option
```

Comment:
nit: I'm blinkering out some other Rubocop whinges here, but it would be nice if these were still aligned.